pyfolio/perf_attrib.py

```diff
@@ -161,6 +180,15 @@ def show_perf_attrib_stats(returns, positions, factor_returns,
 def plot_returns(returns, specific_returns, common_returns, ax=None):
     """
     Plot total, specific, and common returns.
+
+    Parameters
```

Comment:
Also document `returns`, `specific_returns` and `common_returns`

Reply:
fixed